File: javaScript/Task/StrogNumberCheck.js
 

let number = 6;

let sum = 0;
let temp = number;

for(; temp != 0; temp /= 10){
	let num = temp % 10;
	let factoria = 1;
	
	for(let count = 1; count <= num; count++){
		sum += factoria;
		
	}

}

if(sum == number){
console.log(number + " is strong Number");

}else
	console.log(number + " not a strong number");

File: java/classWork/StrogNumberCheck.java
import java.util.Scanner;

public class StrogNumberCheck{
public static void main(String...ekwe){

Scanner input = new Scanner(System.in); 

System.out.print("Enter a number: ");
int number = input.nextInt();

int sum = 0;
int temp = number;

for(; temp != 0; temp /= 10){
	int num = temp % 10;
	int factoria = 1;
	
	for(int count = 1; count <= num; count++){
		sum += factoria;
		
	}

}
if(sum == number){
System.out.print(number + " is perfect Number");

}else
	System.out.print(number + " not a perfect number");

}


}